### ISSUE TITLE
Island Fly Addon > Keep island fly on after leaving your island.

### ISSUE THREAD

**Issue by MamaBeau** [BentoBoxWorld/IslandFly#34]
#### Is your feature request related to a problem?

Hello, I am here to request that when you leave an island and have /is fly enabled that it remain turned on when you leave your island to go to another world, or another players island.
Currently, the island fly will on stay enabled when logging off/on or on server restart.

#### Describe the solution you'd like us to implement.

When leaving an island (at least on my server) your island fly is disabled when you come back to your island and you then have to type /is fly again. I believe it would be much more appealing to have this island fly stay toggle on no matter what, so it doesn't turn off unless you toggle /is fly 

#### Describe alternatives you've considered.

There's not really an alternative solution here.

**Comment by tastybento** [comment]
Okay, just to clarify, this is a request that if you have fly on and then go away from your island then it _remembers_ that you had fly on when you return and resumes it. I originally read this as a request to keep flying on even if you were on another player's island or another world.

**Comment by MamaBeau** [comment]
You are correct, I would like it to remember the fly when leaving the island

**Comment by MamaBeau** [comment]
I would really appreciate this, along with safe login too just for being on the island. So when you log out of the server and back in, it remembers that you are using island fly. 

Obviously essentials and CMI have this feature for their flying, however the player having this actually will allow them to fly anywhere on the server.
